### ISSUE TITLE
document bullmq support

### ISSUE THREAD

**Issue by ankitjena** [bee-queue/arena#248]
## Description
I was trying to use `bull-arena` for my project which uses `bullmq`. The project is written in typescript.
Here's what I am trying to do
```
import Arena from "bull-arena";
import { Queue } from "bullmq";

const arena = Arena({
  Bull: Queue,
  queues: [
    {
      name: "link-account-queue",
      hostId: "Link Account Queue",
      redis: {
        host: REDIS_HOST,
        port: REDIS_PORT
      }
    }
  ]
});
```
Typescript throws an error saying
```
Type 'typeof Queue' is not assignable to type '{ <T = any>(queueName: string, opts?: QueueOptions): Queue<T>; <T = any>(queueName: string, url: string, opts?: QueueOptions): Queue<T>; new <T = any>(queueName: string, opts?: QueueOptions): Queue<...>; new <T = any>(queueName: string, url: string, opts?: QueueOptions): Queue<...>; }'.
  Type 'typeof Queue' provides no match for the signature '<T = any>(queueName: string, opts?: QueueOptions): Queue<T>'
```
If i force the type to never, I can start the application but I get the following error when opening the arena ui. 
```
TypeError: queue.client.info is not a function
```

## Question
- Does `bull-arena` support `bullmq` with `typescript`?
- If so, what am I doing wrong and what are the workarounds?

Can someone help me out. Thanks in advance

**Comment by skeggse** [comment]
@gtpan77 do you have any ideas about this?

**Comment by gtpan77** [comment]
No, It doesn't support bullmq but we should support it soon bcoz `bullmq is next major version of Bull`.
@ankitjena It may work if you use `require` bullmq queue instead of `import`.

**Comment by skeggse** [comment]
oh lol I didn't even notice that wasn't just bull

**Comment by gtpan77** [comment]
Assign this issue to me. I'll look into it.

**Comment by skeggse** [comment]
Support landed in #251 (with typescript support in DefinitelyTyped/DefinitelyTyped#48886). Thanks, @gtpan77!

Documentation still needed, though. I'm going to repurpose this issue to track that work. Take bullmq for a spin in v3.4.0 and let us know how it goes!

**Comment by skeggse** [comment]
Documented in #273, thanks @gtpan77!
